clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/auth/DockerBearerTokenService.groovy

```diff
@@ -67,6 +69,17 @@ class DockerBearerTokenService {
       resolvedPassword = "" // I'm assuming it's ok to have an empty password if the username is specified
     }
 
+    if (resolvedPassword?.length() > 0) {
+      def message = "Your registry password has %s whitespace, if this is not intentional authentication will fail."
+      if (resolvedPassword.charAt(0).isWhitespace()) {
```

Comment:
We talked about this offline - but `[0]` seems to return a `String` rather than a `char` likely due to the fact that groovy supports that fancy range syntax.